docs/release-process.md

```
@@ -98,19 +100,20 @@ selector:
 ```
 
-* Create the registry image. (Below command assumes the release version as `v0.2.0`; please change the command accordingly.)
-  
-```txt
-  make registry-build REGISTRY_IMG=quay.io/argoprojlabs/argocd-operator-registry:v0.2.0-rc1
-```
-
-* Push the registry image. (Below command assumes the release version as `v0.2.0`; please change the command accordingly.)
+* Build and push the registry image. (Below commands assume the release version as `v0.2.0`; please change the command accordingly.)
   
 ```txt
```

markdownlint-cli2 (0.23.2)

[warning] Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)